dashboard/modules/job/cli.py

```diff
@@ -19,10 +20,21 @@
 
 def _get_sdk_client(
     address: Optional[str],
+    headers: Optional[str],
     create_cluster_if_needed: bool = False,
     verify: Union[bool, str] = True,
 ) -> JobSubmissionClient:
-    client = JobSubmissionClient(address, create_cluster_if_needed, verify=verify)
+
+    if address is None and "RAY_ADDRESS" in os.environ:
+        address = os.environ["RAY_ADDRESS"]
+
+    print("YOOO")
+    if headers is None and "RAY_JOB_HEADERS" in os.environ:
+        print("YO")
+        headers = os.environ["RAY_JOB_HEADERS"]
```

Comment:
It looks like `RAY_JOB_HEADERS` is only being read in the CLI codepath. However, some users use the SDK directly.  They may expect that if `RAY_JOB_HEADERS` is set directly, it will be picked up by their `JobSubmissionClient.` Can you fix this? Alternatively, we can defer the `RAY_JOB_HEADERS` to a subsequent PR and just focus on the CLI argument `--headers` in this PR.

Reply:
Let's defer it, and we can use both!